ci/Dockerfile

```diff
@@ -17,31 +17,29 @@
 ###
 # Build the image for spark-rapids-jni development environment.
 #
-# Arguments: CUDA_VERSION=11.8.0
+# Arguments: CUDA_VERSION=[11.X.Y, 12.X.Y], OS_RELEASE=[8, 9], TARGETPLATFORM=[linux/amd64, linux/arm64]
 #
 ###
 ARG CUDA_VERSION=11.8.0
-FROM nvidia/cuda:$CUDA_VERSION-devel-centos7
-ARG DEVTOOLSET_VERSION=11
+ARG OS_RELEASE=8
+ARG TARGETPLATFORM=linux/amd64
+# multi-platform build with: docker buildx build --platform linux/arm64,linux/amd64 <ARGS> on either amd64 or arm64 host
+# check available official arm-based docker images at https://hub.docker.com/r/nvidia/cuda/tags (OS/ARCH)
+FROM --platform=$TARGETPLATFORM nvidia/cuda:$CUDA_VERSION-devel-rockylinux$OS_RELEASE
```

Comment:
Do we need to specify a platform here?  I would expect Docker to automatically select a base image compatible with the current processor architecture (e.g.: we didn't have to specify this before when using other OS base images available on multiple architectures).  If we're always going to build on the platform we expect to run on, I'm not sure why we need to explicitly state the platform here and in the build command.

Reply:
We have the CI jobs to cross-build the arm64 JNI images on the x86_64 Blossom machines.

`--platform=$TARGETPLATFORM` is set to be compatible with docker cross-building docker cases.